test/example/scene_example.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Example do
+  @moduledoc """
+  This module covers two examples for scene API.
+
+  An easy example simulates the layout of a video conferencing app.
+  There are 4 video inputs. All of them get their corners rounded. Then,
+  one of them is chosen as a main video (which takes most of the screen),
+  while the rest are scaled-down and put in a side strip on top of the main video.
+
+  A hard example takes three videos, applies varied transformations to them, and puts
+  them together on the final canvas
+  - The first video is simply rotated and then put in the top left corner
+  - The second video is merged with the first video. The result gets
+  corner rounding and is put in the top right corner
+  - The third video is turned into a ball and put in the middle bottom of the screen
+  """
+
+  alias Membrane.VideoCompositor.{Canvas, Compound, Scene, Texture}
+  alias Membrane.VideoCompositor.Compound.BaseSize
+  alias Mock.{CornerRounding, Grid, Merging, Overlay, Rotate, ToBall}
+
+  @easy %Scene{
+    alternations: [
+      rounding: %CornerRounding{pixels: 10, degrees: 60}
+    ],
+    layouts: [
+      overlay: Overlay,
+      strip: Grid
+    ],
+    objects: [
+      rounded_1: %Texture{input: :video_1, transformations: [:rounding]},
+      rounded_2: %Texture{input: :video_2, transformations: [:rounding]},
+      rounded_3: %Texture{input: :video_3, transformations: [:rounding]},
+      rounded_4: %Texture{input: :video_4, transformations: [:rounding]},
+      strip: %Compound{
+        base_size: %BaseSize{height: 1080, width: 480},
+        inputs_map: %{
+          rounded_1: :top,
+          rounded_2: :middle,
+          rounded_3: :bottom
+        },
+        layout: :strip
+      },
+      final_object: %Compound{
+        base_size: %BaseSize{height: 1080, width: 1920},
+        inputs_map: %{
+          strip: 1.0,
+          rounded_4: 0.0
+        },
```

Comment:
I'm a bit unsure about this `inputs_map` thing, but I don't have any better ideas.